app/mailers/notifier.rb

```diff
@@ -28,7 +28,8 @@ def single_admin(string, recipient, opts={}, subject=nil)
 
     default_opts = {:to => @receiver.email,
          :from => AppConfig.mail.sender_address,
-         :subject => subject, :host => AppConfig.pod_uri.host}
+         :subject => subject, :host => AppConfig.pod_uri.host,
+      :sender => AppConfig.mail.sender_address}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.